chart/epinio-installer/templates/manifest.yaml

```
@@ -10,6 +10,11 @@ stringData:
     {{- $registryPassword := default (randAlphaNum 10) .Values.externalRegistryPassword -}}
     {{- $accessKey := randAlphaNum 16 -}}
     {{- $secretKey := randAlphaNum 16 -}}
+    {{- $oldkeys := (lookup "v1" "secret" "minio-epinio" "tenant-creds").data -}}
```

```suggestion
    {{- $oldkeys := (lookup "v1" "Secret" "minio-epinio" "tenant-creds").data -}}
```

Lettercase matters for type. Note, during testing switched from if/else to use `ternary`.